test/handlers/FranchiserFactoryHandler.sol

```diff
@@ -167,6 +203,30 @@ contract FranchiserFactoryHandler is Test {
         sum = delegators.reduce(acc, _getAccountBalance);
     }
 
+    // function to validate the account balances in the mapping match the actual balances
+    function validateAccountBalances() public view returns (bool valid) {
+        valid = true;
+        for (uint256 i = 0; i < accountBalanceAddresses.length(); i++) {
```

Comment:
in other places where we do this, we use the `AddressSet.forEach` method to iterate. nit here is that i'd prefer consistency across codebases when possible -- invariant suites are already hard enough to grok. to accomplish this we'd have to add `forEach` to EnumerableSet. then, we'd need a `handler.forEachAccount` and we'd need to pass a method from the invariant file into this (aka `this.assertAccountBalance`). I think this is more idiomatic, plus would articulate the assertion more clearly (with offending case logged explicitly). See example link in Element.

Reply:
Done: 66e21be728f196ef3bb5ff940465124e7fa45d0d